### DISCUSSION TITLE
Can a DVD drive be passed into an Ubuntu VM?

### DISCUSSION THREAD

**Discussion by norman350** [quickemu-project/quickemu#1316] [Quickemu]
Just curious if the host DVD drive can be passed into the guest VM (Ubuntu).  Thanks!

**Comment by flexiondotorg** [comment]
Do you mean a physical drive?

**Reply by norman350** [reply]
Yes.  Host device /dev/sr0

brw-rw----+ 1 root cdrom 11, 0 Jun 25 10:42 /dev/sr0

**Comment by flexiondotorg** [comment]
No, this is not something `quickemu` currently supports. `qemu` can do it but I've not exposed this capability via `quickemu`. Perhaps using `--extra_args <arguments>` of `quickemu` could be utilised to pass additional arguments to `qemu`?

Out of interest, what do you want to use this feature for?

**Reply by norman350** [reply]
I have data on dvd I wanted to read directly in VM.  I'll look into extra args or network shares.  I was just looking for the easy button. thanks for the info.  I appreciate all the work you have put into this project!
